### PR TITLE
Guideline for variable naming convention is done

### PR DESCRIPTION
**Pull request by Ayush8923**
Targets #1003

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments below:
- There is no formatting in the document. Everything is written in plain text. 
- The file should be inside the docs folder
- The link to this file should be added in the [development.md](https://github.com/ColoredCow/portal/blob/master/docs/development.md) file as `Variable Naming Convention in Employee Portal` under the section `Coding Guidelines and Conventions`.
- We don't need to add all the examples of how to access an element using id name, class, name etc. Only the variable names in the proper cases will work. And also there is no need to define what Camecase, Titlecase is.

**Review by Ayush8923 — COMMENTED** [review]
If you have changed according to the comment, please review it.

File: docs/variable-naming-convention.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Guideline For Variable Naming Convention
+
+### JavaScript:
+className - snake-case
+idName - camelCase
```
Comment (GauravGusain98) [inline docs/variable-naming-convention.md:5]:
```suggestion
class name - snake-case
id name - camelCase
```

File: docs/variable-naming-convention.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### PHP:
+variable name - camelCase
+function name - camelCase
+class name - TitleCase
+
+### Blade:
+variable name - camelCase
+className - snake-case
+idName - camelCase
```
Comment (GauravGusain98) [inline docs/variable-naming-convention.md:15]:
```suggestion
class name - snake-case
id name - camelCase
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor suggestions. Please check.

**Comment by Ayush8923** [comment]
Some changes are done, please review it.

File: docs/variable-naming-convention.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+### PHP:
+variable name - camelCase
+function name - camelCase
+class name - TitleCase
+
+### Blade:
+variable name - camelCase
+class name - snake-case
+id name - camelCase
```
Comment (GauravGusain98) [inline docs/variable-naming-convention.md:15]: It's not looking correctly in Display mode
image


Please make the changes as provided below
```suggestion
### JavaScript:
- Class name - snake-case
- Id name - camelCase

### PHP:
- Variable name - camelCase
- Function name - camelCase
- Class name - TitleCase

### Blade:
- Variable name - camelCase
- Class name - snake-case
- Id name - camelCase
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**